pkg/noderesourcetopology/filter_test.go

```diff
@@ -586,6 +586,31 @@ func TestNodeResourceTopologyMultiContainerPodScope(t *testing.T) {
 	}
 }
 
+const (
```

Comment:
reading the code down below it seems we save little, coming from
```
state: "Pending"
```
to
```
state: pending
```

Reply:
It provides 2 small (but nice, consider the effort) benefits:
1. In case of future name changes - change once
2. You are getting self-completion on modern IDEs, which make it faster and help you avoid typoes.

Reply:
I don't have strong feelings here so let's keep it

Reply:
If we states we care about are only 2 (and this seems the case) I wonder if we can abstract even further and just use a simple `running bool` and infer the pending state automatically